### PR TITLE
Add workflow for docs spellchecker

### PR DESCRIPTION
**Pull request by paescuj**
## Description

Add workflow for docs spellchecker & skip core checks on pure docs changes.

Part of [ENG-876](https://linear.app/directus/issue/ENG-876/adjust-github-workflows-for-docs)

## Type of Change

- [ ] Bugfix
- [ ] Improvement
- [ ] New Feature
- [x] Refactor / codestyle updates
- [ ] Documentation
- [ ] Other, please describe:

## Requirements Checklist

- [ ] New / updated tests are included
- [x] All tests are passing locally
- [x] Performed a self-review of the submitted code

If adding a new feature:

- [ ] Documentation was added/updated

### CODE REVIEW

File: .github/workflows/docs.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+jobs:
+  spellcheck:
+    name: Check Spelling
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Prepare
+        uses: ./.github/actions/prepare
```
[the remark this answers is not in the document]
Comment (paescuj) [reply .github/workflows/docs.yml:30]: Nice catch! 😃

File: .github/workflows/check.yml [hunk 2]
```diff
@@ -4,9 +4,13 @@ on:
   push:
     branches:
       - main
+    paths-ignore:
+      - docs/**
```
[the remark this answers is not in the document]
Comment (paescuj) [reply .github/workflows/check.yml:8]: Good thought! But in this case I tend not to define too many exceptions and rather run the workflow once too often.

**Review by rijkvanzanten — APPROVED** [review]

File: .github/workflows/docs.yml
[the remark this answers is not in the document]
Comment (azrikahar) [reply .github/workflows/docs.yml:1]: Thanks for clarifying!

[the remark this answers is not in the document]
Comment (paescuj) [reply .github/workflows/docs.yml:1]: Thanks, had the same idea 😄👍 Although, I thought about creating a workflow for the whole code base which is configured to only check the files which have changed in the current commit. I think we should address this in a separate pull request - for now it's enough to have the same checks as before in the dedicated repo.

> (on a slight off-topic, is this root-level "format" script actually used in any current workflows? 🤔)

No, but `prettier` rules are checked via `eslint` for all JS/TS files 👍

**Pull request merged**